bin/cargo-tessel.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+// Helper tool for creating cross-compiled bundles for Tessel deployment.
+
+// System Objects
+// ...
+
+// Third Party Dependencies
+var parser = require('nomnom');
+
+// Internal
+var log = require('../lib/log');
+var rust = require('../lib/install/rust');
+
+function die(err) {
+  log.error(err.stack);
+  process.exit(1);
+}
```

Comment:
We should refactor this to use `closeSuccessfulCommand` and `closeFailedCommand`, but I won't block this PR on that.